internal/digraph/executor/mail.go

```diff
@@ -93,11 +96,49 @@ func (e *mail) Run(ctx context.Context) error {
 		return fmt.Errorf("no valid recipients specified")
 	}
 
-	_, _ = fmt.Fprintf(e.stdout, mailLogTemplate, e.cfg.From, strings.Join(toAddresses, ", "), e.cfg.Subject, e.cfg.Message)
+	var ccAddresses []string
+	switch v := e.cfg.Cc.(type) {
+	case string:
+		if v != "" {
+			ccAddresses = []string{v}
+		}
+	case []string:
+		ccAddresses = v
+	case []any:
+		for _, addr := range v {
+			if str, ok := addr.(string); ok && str != "" {
+				ccAddresses = append(ccAddresses, str)
+			}
+		}
+	default:
+		return fmt.Errorf("invalid type for 'cc' field: expected string or array, got %T", v)
+	}
+
+	var bccAddresses []string
+	switch v := e.cfg.Bcc.(type) {
+	case string:
+		if v != "" {
+			bccAddresses = []string{v}
+		}
+	case []string:
+		bccAddresses = v
+	case []any:
+		for _, addr := range v {
+			if str, ok := addr.(string); ok && str != "" {
+				bccAddresses = append(bccAddresses, str)
+			}
+		}
+	default:
+		return fmt.Errorf("invalid type for 'bcc' field: expected string or array, got %T", v)
+	}
```

Comment:
This code seems a bit repetitive. Could we consider to define an utility function to parse these address fields, something like below?
```go
func parseAddressField(value any) ([]string, error)
```